interfaces/cython/cantera/_onedim.pyx

```diff
@@ -1523,7 +1523,7 @@ cdef class Sim1D:
             return self.sim.fixedTemperatureLocation()
 
     def save(self, filename='soln.yaml', name='solution', description=None,
-             loglevel=1, compression=0):
+             loglevel=None, overwrite=False, compression=0):
```

Comment:
I think you can avoid breaking code for anyone using positional arguments here by moving `overwrite` to the end. It'd be unfortunate if someone misses the warnings and accidentally overwrites a file due to positive integers being truthy. Minor concern, but I hope it's not a big lift to change 🤓

Reply:
Compression is a very new feature and was never released, so I believe this is ok. I was more concerned about loglevel being caught.

Reply:
Oh, ok. This is what I get for commenting when I haven't been paying attention to the changes! You're right, this probably makes sense then the way it is.